llvm-spirv/lib/SPIRV/SPIRVUtil.cpp

```diff
@@ -288,7 +288,7 @@ bool isSYCLHalfType(llvm::Type *Ty) {
       return false;
     StringRef Name = ST->getName();
     Name.consume_front("class.");
-    if ((Name.startswith("cl::sycl::") ||
+    if ((Name.startswith("cl::sycl::") || Name.startswith("sycl::") ||
```

Comment:
This is being implemented in https://github.com/KhronosGroup/SPIRV-LLVM-Translator/pull/1564

I expect the review to take some time, so including it here to get testing results and allow people to remove other places.

Reply:
I think this is reasonable. Though I see it merged, it might be a while before it gets pulled in.

Reply:
My original comment was slightly misleading, I've edited it. I'm currently aiming to have this review finished about the same time the fix gets pulled in through normal processes.

Sorry for any confusion.